pod/perlunicook.pod

```diff
@@ -696,17 +695,15 @@ When run, the following program produces this nicely aligned output:
     寿司............... €9.99
     包子............... €7.50
 
-Here's that program; tested on v5.14.
+Here's that program.
 
  #!/usr/bin/env perl
  # umenu - demo sorting and printing of Unicode food
  #
  # (obligatory and increasingly long preamble)
  #
+ use v5.36;
```

Comment:
This code uses prototypes, so this change would break it.

Reply:
Ugh. :( How annoying.

I wonder if I should revert it, or fix it up with `:proto(...)` attributes. Any preference?

Reply:
Fixed 5a03b3b9b5